styles/_notifications.scss

```diff
@@ -0,0 +1,151 @@
+.notification {
```

Comment:
Can we add a `/***** Notifications *****/` comment at the top of this partial so it segments this section in the compiled file?

Reply:
Amended